docs/tutorials/AddRemark.md

```diff
@@ -3,16 +3,16 @@ layout: page
 title: "Tutorial: Adding a command"
```

Comment:
Should we remove all the tutorials?

Reply:
I agree, but i think not super important. Can do when we have time